modules/core/client/components/Map/index.js

```diff
@@ -30,6 +31,21 @@ export default function Map(props) {
     zoom,
   });
 
+  useEffect(() => {
+    const debouncedHandleResize = debounce(function handleResize() {
```

Comment:
[`debounce`](https://lodash.com/docs/4.17.15#debounce) is to ensure we don't refresh constantly while refreshing, thus avoiding performance issue.

Reply:
Could we even afford [`throttle`](https://lodash.com/docs/#throttle) and shorter time?
I think 1 second is quite long.